lib/engine/game/g_1867/game.rb

```diff
@@ -463,8 +463,16 @@ def maximum_loans(entity)
 
         def home_token_locations(corporation)
           # Can only place home token in cities that have no other tokens.
+          # Minors can go in a disconnected Toronto/Montreal station, but Majors
+          # cannot.
           open_locations = hexes.select do |hex|
-            hex.tile.cities.any? { |city| city.tokenable?(corporation, free: true) && city.tokens.none? }
+            case corporation.type
+            when :minor
+              hex.tile.cities.any? { |c| c.tokenable?(corporation, free: true) && c.tokens.none? }
+            when :major
+              hex.tile.cities.any? { |c| c.tokenable?(corporation, free: true) } &&
+                hex.tile.cities.all? { |c| c.tokens.none? { |t| t&.type == :normal } }
```

Comment:
I read Ian's ruling differently. I thought he said majors start in an unconnected city, unless none of those exist, in which case, they can start in a connected city, provided no tokens are in that city.

Reply:
Or does this only address not starting two majors in the same tile? In which case, do you need to check if the token is from another major (or CN)?

Reply:
> I read Ian's ruling differently. I thought he said majors start in an unconnected city, unless none of those exist, in which case, they can start in a connected city, provided no tokens are in that city.

That's correct. This change is to the `open_locations` variable, which gives all cities which could potentially be used as a starting location. A few lines below (line 480 in the modified file) this array is filtered to return only the cities which are unconnected, if there are any.

Reply:
> Or does this only address not starting two majors in the same tile? In which case, do you need to check if the token is from another major (or CN)?

No, you can't start the major on the tile if there are any other tokens on the tile. It doesn't matter whether they are owned by another major, a minor, or the national.